docs/whats-new/dotnet-docs-mod0.md

```diff
@@ -1,118 +1,108 @@
-- [Assembly validation](../fundamentals/apicompat/assembly-validation.md)
-- [API compatibility tools](../fundamentals/apicompat/overview.md)
-- [Microsoft.Testing.Platform exit codes](../core/testing/unit-testing-platform-exit-codes.md)
+- [Access custom attributes](../fundamentals/reflection/accessing-custom-attributes.md)
+- [Dynamically load and use types](../fundamentals/reflection/dynamically-loading-and-using-types.md)
+- [Emit dynamic methods and assemblies](../fundamentals/reflection/emitting-dynamic-methods-and-assemblies.md)
+- [How to: Define a generic method with reflection emit](../fundamentals/reflection/how-to-define-a-generic-method-with-reflection-emit.md)
+- [How to: Define a generic type with reflection emit](../fundamentals/reflection/how-to-define-a-generic-type-with-reflection-emit.md)
+- [How to: Define and execute dynamic methods](../fundamentals/reflection/how-to-define-and-execute-dynamic-methods.md)
+- [How to: Examine and instantiate generic types with reflection](../fundamentals/reflection/how-to-examine-and-instantiate-generic-types-with-reflection.md)
+- [How to: Hook up a delegate using reflection](../fundamentals/reflection/how-to-hook-up-a-delegate-using-reflection.md)
+- [Reflection and generic types](../fundamentals/reflection/reflection-and-generic-types.md)
+- [View type information](../fundamentals/reflection/viewing-type-information.md)
+- [Upgrade to a new .NET version](../core/install/upgrade.md)
+- [CA1872: Prefer 'Convert.ToHexString' and 'Convert.ToHexStringLower' over call chains based on 'BitConverter.ToString'](../fundamentals/code-analysis/quality-rules/ca1872.md)
+- [MSTEST0017: Assertion arguments should be passed in the correct order](../core/testing/mstest-analyzers/mstest0017.md)
+- [MSTEST0019: Prefer TestInitialize methods over constructors](../core/testing/mstest-analyzers/mstest0019.md)
+- [MSTEST0020: Prefer constructors over TestInitialize methods](../core/testing/mstest-analyzers/mstest0020.md)
+- [MSTEST0021: Prefer Dispose over TestCleanup methods](../core/testing/mstest-analyzers/mstest0021.md)
+- [MSTEST0022: Prefer TestCleanup over Dispose methods](../core/testing/mstest-analyzers/mstest0022.md)
+- [MSTEST0023: Do not negate boolean assertions](../core/testing/mstest-analyzers/mstest0023.md)
+- [MSTEST0024: Do not store TestContext in a static member](../core/testing/mstest-analyzers/mstest0024.md)
+- [MSTest overview](../core/testing/unit-testing-mstest-intro.md)
 - [Microsoft.Testing.Platform overview](../core/testing/unit-testing-platform-intro.md)
-- [Microsoft.Testing.Platform telemetry](../core/testing/unit-testing-platform-telemetry.md)
+- [Use runsettings with Microsoft.Testing.Platform](../core/testing/unit-testing-platform-runsettings.md)
 - [Microsoft.Testing.Platform and VSTest comparison](../core/testing/unit-testing-platform-vs-vstest.md)
+- [MSTEST0016: Test class should have test method](../core/testing/mstest-analyzers/mstest0016.md)
```

Comment:
Not sure why these aren't grouped with the other MSTEST rules.

Reply:
I haven't had the chance to implement that sort yet. Hopefully this month.